packages/cli/mock/src/runMockServer.ts

```diff
@@ -0,0 +1,208 @@
+import { assertNever } from "@fern-api/core-utils";
+import { generateIntermediateRepresentation } from "@fern-api/ir-generator";
+import { TaskContext } from "@fern-api/task-context";
+import { FernWorkspace } from "@fern-api/workspace-loader";
+import {
+    ExampleEndpointCall,
+    ExamplePathParameter,
+    ExampleTypeReference,
+    HttpEndpoint,
+    NameAndWireValue
+} from "@fern-fern/ir-sdk/api";
+import express, { Request, Response } from "express";
+import { ParamsDictionary, Query } from "express-serve-static-core";
+import getPort from "get-port";
+import { IncomingHttpHeaders } from "http";
+import { isEqual } from "lodash-es";
+import qs from "qs";
+import urlJoin from "url-join";
+
+type RequestHandler = (req: Request, res: Response) => void;
+
+export async function runMockServer({
+    context,
+    workspace,
+    port
+}: {
+    context: TaskContext;
+    workspace: FernWorkspace;
+    port: number | undefined;
+}): Promise<void> {
+    const ir = await generateIntermediateRepresentation({
+        workspace,
+        audiences: { type: "all" },
+        generationLanguage: undefined
+    });
+
+    const app = express();
+    app.use(express.json({ limit: "50mb", strict: false }));
+
+    for (const service of Object.values(ir.services)) {
+        for (const endpoint of service.endpoints) {
+            const endpointPath = getFullPathForEndpoint(endpoint);
+            context.logger.debug(`Registering ${endpoint.method} ${endpointPath} ...`);
```

Comment:
i could see this being info actually

Reply:
I'm OK with starting with `info`, but I figured this would end up looking like a massive wall of text for larger APIs. We can always change it to `debug` later though, so I'm fine with changing this here.

Reply:
yeah i think we could always limit to 25 paths or something but kind of nice to see all the paths that the server is serving